src/components/Common/BlogListing.js

```diff
@@ -15,18 +29,20 @@ const BlogListing = ({ title, description, posts }) => {
   }))
 
   return (
-    <Grid>
-      <Padding vertical={{ desktop: 5, smallPhone: 3.5 }}>
-        <TitleAndMediaList
-          title={title}
-          description={description}
-          mediaItems={mediaItems}
-          CTALink="/blog/"
-          external
-          CTAText="Visit our blog"
-        />
-      </Padding>
-    </Grid>
+    <Wrapper bgColor={blogBackgroundColor}>
+      <Grid>
+        <Padding vertical={{ desktop: 5, smallPhone: 3.5 }}>
```

Comment:
Can we put the Padding component above the Grid? That way we could keep the Grid > Row > Col scheme

Reply:
Yeah, this is annoying as the TitleAndMediaList component has `RowLayout` internally. I would opt for removing it altogether but not sure where to put the padding instead as I don't really feel it should be on the grid